data/upstream.yaml

```diff
@@ -215,6 +215,11 @@ gobject-introspection:
   links:
     homepage: https://wiki.gnome.org/Initiatives/GnomeGoals/Python3Porting
     bug: https://bugzilla.gnome.org/show_bug.cgi?id=679438
+graphviz:
+  status: released
+  links:
+    homepage: https://pypi.python.org/pypi/graphviz
+    repo: https://github.com/xflr6/graphviz
```

Comment:
You are right, the same name made me a confusion. I revert the commit, and make homepage/repo appropriate update to avoid the confusion for future.